.github/workflows/docker.yml

```
@@ -0,0 +1,18 @@
+name: Docker
+on:
+  pull_request:
+    paths:
+      - .github/workflows/docker.yml
+      - Dockerfile
+jobs:
+  ubuntu:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build Docker image
+        run: docker build -t homebrew/debian7 .
+      - name: Deploy the Docker image to Docker Hub
+        if: startsWith(github.ref, 'refs/tags/')
+        run: |
+          docker login -u brewtestbot -p ${{secrets.DOCKER_TOKEN}}
+          docker push "homebrew/homebrew/debian7:latest"
```

```suggestion
          docker push homebrew/debian7:latest
```

I think it should be like that.